src/site/includes/breadcrumbs.drupal.liquid

```diff
@@ -1,52 +1,17 @@
 <nav aria-label="Breadcrumb" aria-live="polite" class="va-nav-breadcrumbs" id="va-breadcrumbs">
   <ul class="row va-nav-breadcrumbs-list columns" id="va-breadcrumbs-list">
-    {% if path == "health-care" or path == "drupal/health-care" %}
-      <li>
-        <a href="/">
-          Home
-        </a>
-      </li>
-      <li>
-        <a href="/{{path}}" aria-current="page">
-          Health Care
-        </a>
-      </li>
-    {% else %}
-      {% assign numCrumbs = entityUrl.breadcrumb | size %}
-      {% assign crumbCount = 1 %}
-      {% for crumb in entityUrl.breadcrumb %}
-        {% if crumbCount == numCrumbs %}
-        <li>
-          <a href="/{{path}}" aria-current="page">
-            {{crumb.text}}
-          </a>
-        </li>
-        {% else %}
-          {% if crumb.text == "Health care" %}
-            {% assign crumbPath = "/health-care" %}
-          {% else %}
-            {% assign crumbPath = crumb.url.path %}
-          {% endif %}
-          {% if crumb.text != "Get benefits" and crumb.text != "Manage benefits" and crumb.text != "More resources" %}
-        <li>
-          {% if crumbPath != empty %}
-          <a href="{{crumbPath}}"
-            {% if crumb.url.path == "/" %}
-            onClick="recordEvent({ event: 'nav-breadcrumb', 'nav-breadcrumb-section': 'home' });"
-            {% elsif crumb.text == lastCrumb %}
-            aria-current="page"
+    {% for crumb in entityUrl.breadcrumb %}
+        {% if forloop.last == true %}
+            <li><a aria-current="page" href="{{ crumb.url.path }}">{{ crumb.text }}</a></li>
+        {% elsif crumb.url.path %}
```

Comment:
Is this condition definitely working? Asking because I think the empty string will [evaluate to true](https://help.shopify.com/en/themes/liquid/basics/true-and-false). We also experienced some weirdness with [`empty`](https://github.com/department-of-veterans-affairs/vets-website/pull/9932#discussion_r275595722) yesterday. I'm wondering if this should be `elsif crumb.url.path != blank`.

Reply:
Yeah it's working: successfully not printing a link with an empty href that's in the breadcrumb object. I can change it to !=blank if that's more break-proof tho

Reply:
Actually !=blank doesn't work but the conditional as I have it does.

Reply:
Thank you for confirming!

I wonder if this is an inconsistency with our Liquid compiler, TinyLiquid, versus the actual language spec. Something seems to be off here 🤔